src/algebra/curves/alt_bn128/alt_bn128_g1.cpp

```diff
@@ -257,7 +256,7 @@ alt_bn128_G1 alt_bn128_G1::add(const alt_bn128_G1 &other) const
 alt_bn128_G1 alt_bn128_G1::mixed_add(const alt_bn128_G1 &other) const
 {
 #ifdef DEBUG
-    assert_except(other.is_special());
+    assert(other.is_special());
```

Comment:
We don't compile with ``DEBUG``, I think (based on cursory inspection of depends/packages/libsnark.mk.) But why is this in an ``#ifdef DEBUG`` conditional anyway?